config.py

```diff
@@ -10,6 +10,8 @@
 
 FLASK_HOST = ""
 FLASK_PORT = 5000
+SERVER_BASE = "http://127.0.0.1:5000"
```

Comment:
This is fine as a default, but I will need to be able to set the server base with an environment variable. I don't see where the code is reading `SERVER_BASE` from the environment.

Reply:
It isn't reading from the environment, but you're right that that would be better. I'll change it.

Reply:
Done.